static/bidder-info/seedingAlliance.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "https://b.nativendo.de/cds/rtb/bid?ssp=pbs"
+endpoint: "https://b.nativendo.de/cds/rtb/bid?ssp={{.AccountID}}"
```

Comment:
@sag-henmus  could you explain usage of `account id`.  

Based on account id will request get directed to bidder server hosted in different regions?

Reply:
> @sag-henmus could you explain usage of `account id`.
> 
> Based on account id will request get directed to bidder server hosted in different regions?

@sag-henmus  requesting to address above comment

Reply:
@onkarvhanumante no, this is used as optional identifier for SSPs with a special agreement with us who want to use our adapter.